packages/calendar/test/lion-calendar.test.js

```diff
@@ -272,22 +256,6 @@ describe('<lion-calendar>', () => {
     });
 
     describe('Enabled Dates', () => {
-      it('disables all days before "minDate" property', async () => {
-        const el = await fixture(html`
-          <lion-calendar
-            .selectedDate="${new Date('2000/12/31')}"
-            .minDate="${new Date('2000/12/09')}"
-          >
-          </lion-calendar>
-        `);
-        const elObj = new CalendarObject(el);
-        expect(elObj.getDayObj(15).isDisabled).to.equal(false);
-        elObj.dayEls.forEach((d, i) => {
-          const shouldBeDisabled = i < 8;
-          expect(new DayObject(d).isDisabled).to.equal(shouldBeDisabled);
-        });
-      });
-
       it('disables all days after "maxDate" property', async () => {
```

Comment:
I think something weird is going on here. As I see things, these two tests ( [it disables all days before "minDate" property'](https://github.com/ing-bank/lion/pull/468/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json#diff-ca8bb482e0fdc1421b404fd0dbd774feL275) & [it disables all days after "maxDate" property](https://github.com/ing-bank/lion/pull/468/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json#diff-ca8bb482e0fdc1421b404fd0dbd774feL291) ) go hand in hand. 

At first glance, it looks to me that there is a problem with:

```js
      it('disables all days before "minDate" property', async () => {
        const el = await fixture(html`
          <lion-calendar
            .selectedDate="${new Date('2000/12/31')}"
            .minDate="${new Date('2000/12/09')}"
          >
          </lion-calendar>
        `);
        const elObj = new CalendarObject(el);
        expect(elObj.getDayObj(15).isDisabled).to.equal(false);
        elObj.dayEls.forEach((d, i) => {
          const doHelper = new DayObject(d);
          // These two values are different?!
          expect(doHelper.isDisabled).to.equal(doHelper.el.hasAttribute('aria-disabled'));
          // const shouldBeDisabled = i < 8;
          // expect(new DayObject(d).isDisabled).to.equal(shouldBeDisabled);
        });
      });
```